pict-doc/pict/scribblings/more.scrbl

```diff
@@ -170,6 +170,47 @@ The @racket[style] can include any of the following:
                #:mercury-inset 4)
 ]}
 
+@defproc[(standard-cat
+           [width positive?]
+           [height positive?]
+           [#:left-ear-extent left-ear-extent (>=/c 0)]
+           [#:left-ear-arc left-ear-arc (real-in 0 (* 2 pi))]
+           [#:left-ear-angle left-ear-angle (real-in 0 (* 2 pi))]
+           [#:right-ear-extent right-ear-extent (>=/c 0)]
+           [#:right-ear-arc right-ear-arc (real-in 0 (* 2 pi))]
+           [#:right-ear-angle right-ear-angle (real-in 0 (* 2 pi))]
+           [#:fur-color fur-color (or/c #f string? (is-a?/c color%))]
+           [#:fur-border-color fur-border-color (or/c #f string? (is-a?/c color%))]
+           [#:lip-color lip-color (or/c #f string? (is-a?/c color%))]
+           [#:lip-border-color lip-border-color (or/c string? (is-a?/c color%))]
+           [#:lip-border-width lip-border-width (or/c #f (real-in 0 255))]
+           [#:eye-color eye-color (or/c #f string? (is-a?/c color%))]
+           [#:nose-color nose-color (or/c #f string? (is-a?/c color%))]
+           [#:nose nose pict?]
+           [#:happy? happy? any/c]
+           [#:eyes eyes (or/c #f pict?)]
+           [#:left-eye left-eye (if eyes (or/c #f pict?) pict?)]
+           [#:right-eye right-eye (if eyes (or/c #f pict?) pict?)]
+           [#:whisker-length whisker-length positive?]
+           [#:whisker-droop whisker-droop real?]
+           [#:whisker-width whisker-width (or/c #f (real-in 0 255))]
+           [#:whisker-color whisker-color (or/c string? (is-a?/c color%))]
+           [#:whisker-inset? whisker-inset? any/c]
```

Comment:
What does 'whisker-inset?` do? I wasn't able to see a difference in examples that changed the inset & length, and wasn't sure what would lead to a change after reading part of the implementation.

```
(apply ht-append 40
  (for/list ((inset? '(#f #t)))
    (apply vl-append 40
      (for/list ((len '(10 40 80)))
        (standard-cat 60 60 #:whisker-length len #:whisker-inset? inset?)))))
```